goentri.com.fourthwall.subdomain.json

```diff
@@ -0,0 +1,100 @@
+{
+   "providerId": "goentri.com",
+   "providerName": "Entri",
+   "serviceId": "fourthwall",
+   "serviceName": "Fourthwall",
+   "version": 3,
+   "logoUrl": "https://cdn.goentri.com/logo.svg",
+   "description": "Allows user to easily set up domain using Entri",
+   "syncPubKeyDomain": "goentri.com",
+   "syncRedirectDomain": "api.goentri.com, goentri.com",
+   "variableDescription": "sendgridHost is sendgrid dynamic host, zendeskverification is the text verification value for zendesk, supportHost is the support host being added by fourthwall",
+   "sharedProviderName": true,
+   "records": [
+      {
+         "type": "CNAME",
+         "host": "%sendgridHost%.%supportHost%",
```

Comment:
Same here, `%supportHost%` can be removed in all places (when it's subdomain) or replaced with `@`